<launch>

  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="filters"                        value="pointcloud"/>
    <arg name="align_depth"                      value="true"/>
  </include>

 <node pkg="vision" type="rgb_segmentation.py" name="rgb_segmentation" output="screen" />

  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
    args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/seg_depth/camera_info"/>
    <remap from="image_rect" to="/seg_depth/image_raw"/>
    <remap from="points" to="/rscamera/depth/points"/>
  </node>





</launch>
